net/monitor/inactivity/inactivitymonitor.go

```diff
@@ -36,8 +35,8 @@ func (m *inactivityMonitor) LastActivity() time.Time {
 	return time.Time{}
 }
 
-func CloseClientConn(cc ClientConn) {
-	cc.Close()
+func CloseClientConn(cc ClientConn) error {
```

Comment:
API change, but adding `error` output shouldn't break most code (only cases I can think of is assigning to function pointer with previous signature or adhering to interface).

Reply:
This is concrete default implemenetation. So for default, we can ignore the error.

Reply:
Actually, `func CloseClientConn` is not used anywhere. Do we need it at all? The caller can call `Close` on `CloseClientConn` directly without this wrapper.